assert_writeln_magic.d

```diff
@@ -236,72 +241,69 @@ Tries to lower EqualExpression in AssertExpressions of Unittest blocks to commen
     }
 
     foreach (file; files)
+    {
         if (!ignoredFiles.any!(x => file.name.canFind(x)))
-            file.name.parseFile;
+        {
+            // single files
+            if (inputDir.length == 0)
+                parseFile(file.name, outputDir);
+            else
+                parseFile(file.name, file.name.replace(inputDir, outputDir));
```

Comment:
Tricky! What if `inputDir` is `"."`?

For my own use I have https://github.com/CyberShadow/ae/blob/13bc434ea12c90883f524818510f35970230becb/utils/path.d#L18-L23